src/Components/TicTacToe/Square.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+export default function Square({value, onClick}) {
+  const extendedStyle = {
+    ...style,
+    color: value === "X" ? "red" : "blue",
```

Comment:
Do we later still want to change the X and Y to the cover image of the song?

Reply:
Probably, yes

Reply:
Yes I would say so, I could already test it with a mocked image